test/ex_sdp/attribute/fmtp_test.exs

```diff
@@ -104,6 +104,34 @@ defmodule ExSDP.Attribute.FMTPTest do
       assert {:ok, expected} == FMTP.parse(fmtp)
     end
 
+    test "parses fmtp with sprop-*ps" do
+      fmtp =
+        "96 profile-space=0;profile-id=1;tier-flag=0;level-id=150;interop-constraints=B00000000000;" <>
+          "sprop-vps=QAEMAf//AWAAAAMAAAMAAAMAAAMAlqwJAAAAAQ==;" <>
```

Comment:
Let's put multiple values for one of those sprop sets to make sure we parse it correctly